board/xen/xenguest_arm64_android/Makefile

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier:	GPL-2.0+
+#
+# (C) Copyright 2020 EPAM Systems Inc.
+
+obj-y	:= xenguest_arm64.o
```

Comment:
Do not copy the file, use the existing one: either have a symlink or obj-y	:= ../xenguest_arm64/xenguest_arm64.o